### PR TITLE
Blog: add pre security release announcement

### PR DESCRIPTION
**Pull request by RafaelGSS**
Refs: https://github.com/nodejs-private/security-release/pull/29

cc: @nodejs/tsc

### CODE REVIEW

File: pages/en/blog/vulnerability/july-2024-security-releases.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+---
+
+# Summary
+
+The Node.js project will release new versions of the 22.x, 20.x, 18.x
+releases lines on or shortly after, Tuesday, July 2, 2024 in order to address:
+
+- 3 low severity issues.
+- 1 high severity issues.
+- 2 medium severity issues.
```
Comment (richardlau) [inline pages/en/blog/vulnerability/july-2024-security-releases.md]: Would be good to sort these -- perhaps high severity first?

File: pages/en/blog/vulnerability/july-2024-security-releases.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Impact
+
+The 22.x release line of Node.js is vulnerable to 3 low severity issues, 1 high severity issues, 2 medium severity issues.
+The 20.x release line of Node.js is vulnerable to 3 low severity issues, 1 high severity issues, 2 medium severity issues.
+The 18.x release line of Node.js is vulnerable to 1 high severity issues, 2 medium severity issues.
```
Comment (richardlau) [inline pages/en/blog/vulnerability/july-2024-security-releases.md]: Same re. ordering of severity.

**Review by richardlau — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
We might want to mention that Node.js v22.4.0 includes a security fix for fetch(), already shipped in undici v6.19.2.

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by richardlau — APPROVED** [review]

**Review by marco-ippolito — APPROVED** [review]

**Comment by ovflowd** [comment]
FYI feel free to ignore the failed Vercel check.

But can we hold off the merge of this, until we get the monorepo stuff ready?

**Comment by RafaelGSS** [comment]
> But can we hold off the merge of this, until we get the monorepo stuff ready?

How long would it take? This PR must land today.

**Comment by bmuenzenmeyer** [comment]
> How long would it take? This PR must land today.

it's gonna be longer than that

**Comment by RafaelGSS** [comment]
So I'm afraid that will need to merge it now. Per policy we need to inform our users 7 days before the security release

**Comment by ovflowd** [comment]
You will need to do a forced merge as a TSC member since the main branch is locked right now.

**Comment by RafaelGSS** [comment]
Done

**Pull request merged**